src/plone/api/tests/test_content.py

```diff
@@ -1447,3 +1449,148 @@ def test_get_view_view_not_found(self):
 
         for should_be_there in should_be_theres:
             self.assertIn((should_be_there + "\n"), str(cm.exception))
+
+    def test_get_parents_required_parameter(self):
+        """Test that get_parents requires an obj parameter"""
+
+        with self.assertRaises(MissingParameterError):
+            api.content.get_parents()
+
+    def test_get_parents_basic(self):
+        """Test getting all parents without filter"""
+
+        # Test nested content (in sprint folder)    
+        parents= api.content.get_parents(self.sprint)
+        self.assertEqual(len(parents), 2)
+        self.assertEqual(parents[0].getId(), "events")
+        self.assertEqual(parents[1], self.portal)
+
+        # Test nested content (in team folder)
+        parents= api.content.get_parents(self.team)
+        self.assertEqual(len(parents), 2)
+        self.assertEqual(parents[0].getId(), "about")
+        self.assertEqual(parents[1], self.portal)
+
+    def test_get_parents_with_interface_filter(self):
+        """Test getting all parents with an interface filter"""
+
+        # Test content in nested folder
+        parents= api.content.get_parents(
+            self.sprint,
+            IFolderish
+        )
+
+        # Should return [events, portal] as both are folders
+        self.assertEqual(len(parents), 2)
+        self.assertTrue(IFolderish.providedBy(parents[0]))
+        self.assertTrue(IFolderish.providedBy(parents[1]))
```

Comment:
I would replace this one with:
```suggestion
        self.assertListEqual(len(parents, [parent1, parent2])
```